### PR TITLE
Initial Chanakya Sai Changes 

### PR DESCRIPTION
**Pull request by Chanusai6**
1. here i have these points
2. please commit changes
3. ihebcjv
4. cwjfb
5. cjibe'
6. cnjen
7. njrnv'

### CODE REVIEW

File: 123.jpg
Comment (cgouthami) [inline 123.jpg:1]: please decrease the size of this image file

File: 345.jpg
Comment (cgouthami) [inline 345.jpg:1]: same above

File: index.css [hunk 1]
```diff
@@ -0,0 +1,346 @@
+* {
+  font-family: "Raleway", sans-serif;
+  /* background-color: black; */
+}
+.container {
+  background-color: blue;
+  color: white;
+  aspect-ratio: 16/9;
+}
+/* .imgContainer {
```
Comment (cgouthami) [inline index.css:11]: please remove unnecessary code if you want please comment a proper message

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      <div class="middle">
+        <div class="circle"></div>
+      </div>
+      <div class="bottom"></div>
+    </div> -->
+    <!-- <span class="loader">
+      <span class="ball"></span>
+      <span class="ball"></span>
+      <span class="ball"></span>
+    </span> -->
```
Comment (cgouthami) [inline index.html:81]: same above

File: Dummy/dummy.txt.txt
Comment (cgouthami) [inline Dummy/dummy.txt.txt:1]: please add some content

**Review by cgouthami — COMMENTED** [review]
please check your comments

File: 123.jpg
Comment (Chanusai6) [reply 123.jpg:1]: done

File: 345.jpg
Comment (Chanusai6) [reply 345.jpg:1]: done

File: index.css (hunk 1 above)
Comment (Chanusai6) [reply index.css:11]: done

File: index.html (hunk 2 above)
Comment (Chanusai6) [reply index.html:81]: we need to keep like only as per communication from PO

File: Dummy/dummy.txt.txt
Comment (Chanusai6) [reply Dummy/dummy.txt.txt:1]: done

**Review by cgouthami — APPROVED** [review]
Approved

**Pull request merged**